src/test/database/listRemote.spec.ts

```diff
@@ -0,0 +1,37 @@
+import { expect } from "chai";
+import * as sinon from "sinon";
+import * as nock from "nock";
+import * as utils from "../../utils";
+import * as api from "../../api";
+
+import * as helpers from "../helpers";
+import { RTDBListRemote } from "../../database/listRemote";
+
+describe("ListRemote", () => {
+  const instance = "fake-db";
+  const remote = new RTDBListRemote(instance);
+  const serverUrl = utils.addSubdomain(api.realtimeOrigin, instance);
+  let sandbox: sinon.SinonSandbox;
+
+  beforeEach(() => {
+    sandbox = sinon.createSandbox();
+    helpers.mockAuth(sandbox);
+  });
+
+  afterEach(() => {
+    sandbox.restore();
+    nock.cleanAll();
+  });
+
+  it("should return subpaths from shallow get request", () => {
+    nock(serverUrl)
+      .get("/.json")
+      .query({ shallow: true, limitToFirst: "1234" })
+      .reply(200, {
+        a: true,
+        x: true,
+        f: true,
```

Comment:
Do these always return `true` as the value? Is there a `false` case that's handled differently (or does it not matter)?

Reply:
This just mimics the response payload that the realtime database sends back for a get with shallow=true (there is nothing being returned here -- "true" is just the shorthand value used to represent a pruned JSON subtree).

Reply:
Correct, shallow list always return `x: true`.